InterviewQuestionsTests/Chapters/CustomLinkedLists/CustomLinkedLists.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+using InterviewQuestions.Chapters.CustomLinkedLists;
+
+namespace InterviewQuestionsTests.Chapters.CustomLinkedLists
+{
+    [TestClass]
+    
+    public class CustomLinkedLists
+    {
+        [TestMethod]
+        public void method()
+        {
+            CustomLinkedListsMethods list=new CustomLinkedListsMethods();
+            list.add(15);
+            list.add(11);
+            list.add(12);   
+            list.add(13);
+
+            Assert.AreEqual(list.head.data, 15);
+            Assert.AreEqual(list.head.next.data, 11);
+        }

```

Comment:
Please fix "dereference possible null reference" here.
Also, check other data as well '12,13, null'

check if its null after "13"

Reply:
Assert.AreEqual(list.head.next.next.next.data, 13);
 Assert.AreEqual(list.head.next.next.next.next.next.data, null);
I tried the next data null or not but this test not passed. I can not figure out the problem. 

Also i can not fix "dereference possible null reference".

Reply:
Assert.AreEqual(list.head.next.next.next.next.next.data, null);
Just this line not passed the test.

Reply:
Please consider that you are checking the right thing. Should you check the 'data' of the last item or the item itself?

Reply:
Alright i get it. This is the correct query pattern. Look here bbfa648702f99935f1ad1c331198ac16fd9bf78e

Reply:
I refactored your class name -> 76c914d5a3468e9d9c193c9426b53eb3a1fe9f8a